GitOut/Features/Wpf/Converters/NullOrEmptyToCollapsedConverter.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace GitOut.Features.Wpf.Converters
+{
+    public sealed class NullOrEmptyToCollapsedConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) => !string.IsNullOrEmpty(value as string) ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
Isn't this supported in a Binding using TargetNullValue and FallbackValue? :thinking:

Reply:
it wasn't even used - should have been cleaned up :thumbsup: